tests/testthat/_snaps/expect-setequal.md

```diff
@@ -12,3 +12,31 @@
     * Only in `expected`: 4, 5, 6, 7, 8, 9, 10, 11, 12, ...
     
 
+# expect_contains() gives useful message on failure
+
+    `x1` (`actual`) doesn't contain all the values in `x2` (`expected)`.
+    * Missing from `actual`: "d"
+    * Present in `actual`:   "a", "b", "c"
+    
+
+---
+
+    `x1` (`actual`) doesn't contain all the values in `x3` (`expected)`.
+    * Missing from `actual`: "d", "e"
+    * Present in `actual`:   "a", "b", "c"
+    
+
+# expect_in() gives useful message on failure
+
+    All values of `x1` (`actual`) aren't contained in `x2` (`expected`).
+    * Missing from `expected`: "a"
+    * Present in `expected`:   "b", "c"
```

Comment:
I think this is particularly confusing because you could interpret `All values of` as implying that literally every single element of `x1` isn't contained, but really it is trying to say "at least 1 isn't contained"